packages/web-twig/src/Resources/components/Toast/ToastBarLink.twig

```diff
@@ -0,0 +1,26 @@
+{# API #}
+{%- set props = props | default([]) -%}
+{%- set _color = props.color | default('inverted') -%}
+{%- set _href = props.href -%}
+{%- set _isDisabled = props.isDisabled | default(false) -%}
+{%- set _isUnderlined = props.isUnderlined | default(true) -%}
+
+{# Class names #}
+{%- set _colorClassName = _spiritClassPrefix ~ 'link-' ~ _color -%}
+{%- set _rootDisabledClassName = _isDisabled ? _spiritClassPrefix ~ 'link-disabled' : null -%}
+{%- set _rootUnderlinedClassName = _isUnderlined ? _spiritClassPrefix ~ 'link-underlined' : null -%}
+
+{# Miscellaneous #}
+{%- set _styleProps = useStyleProps(props) -%}
+{%- set _classNames = [ _colorClassName, _rootDisabledClassName, _rootUnderlinedClassName, _styleProps.className ] -%}
+{%- set _allowedAttributes = [ 'target', 'title' ] -%}
+
+<a
```

Comment:
please use link component (if it exists in twig) and pass the default props. I think it was link-inverted and link-underlined. I don't think user will need to change/set these. They should be default and I would be ok if the user won't be able to modify them, when using the ToastBarLink